packages/components/src/core/ButtonIcon/index.stories.tsx

```diff
@@ -195,6 +210,246 @@ export const LivePreview = {
   render: (args: Args) => <LivePreviewDemo {...args} />,
 };
 
+// Screenshot test
+
+const ScreenshotTestDemo = (): JSX.Element => {
+  const topLevel: React.CSSProperties = {
+    columnGap: "20px",
+    display: "inline-grid",
+    fontFamily: "sans-serif",
+    marginRight: "50px",
+  };
+  const displayContents: React.CSSProperties = {
+    display: "contents",
+  };
+  const penultimateLevel: React.CSSProperties = {
+    display: "contents",
+  };
+  const bottomLevel: React.CSSProperties = {
+    marginBottom: 10,
+  };
+  const topLabel: React.CSSProperties = {
+    fontSize: "2em",
```

Comment:
maybe move the constants closer to their first use and rename them with ALL_CAPS. There seems to be a few that can be declared inside their corresponding component functions too!

E.g.,

```tsx
function ButtonIconSizeOption() {
  const SECOND_LABEL = {
    ...midLabel,
    borderWidth: "2px",
    fontSize: "1.17em",
    margin: "20px 0",
  }

  return (
    ...
     <p style={SECOND_LABEL}>
  )
}
```

Also, to improve readability, if the variables are declared at their corresponding component level, perhaps we can even rename them?

For example, currently we have `secondLabel`, which doesn't convey much meaning. Now that the variable is declared inside `<ButtonIconSizeOption />`, perhaps we can rename it to `LABEL_STYLE`?

```tsx
function ButtonIconSizeOption() {
  const LABEL_STYLE = {
    ...midLabel,
    borderWidth: "2px",
    fontSize: "1.17em",
    margin: "20px 0",
  }

  return (
    ...
     <p style={LABEL_STYLE}>
  )
}
```